### PR TITLE
Remove old doc and revamp repo to guides

### PR DESCRIPTION
**Pull request by alasarr**
Remove old documentation: this documentation is not outdated because it was moved to https://codesandbox.io/s/carto-deck-gl-examples-rimo9x

I have also moved the guides to root folder as this repo will be renamed to carto-for-developers-guides

### CODE REVIEW

File: integrate-existing-app/backend/.env.local [hunk 1]
```diff
@@ -0,0 +1,8 @@
+PORT=8000
+# The JWT Secret to sign the JWT token
+JWT_SECRET=PUT_YOUR_SECRET_HERE
+# API Base URL (copy this from CARTO Workspace -> Developers)
+CARTO_API_BASE_URL=https://gcp-us-east1.api.carto.com
+# ClientID and Secret of your Machine to Machine Application
+CARTO_CLIENT_ID=GlLAKY2XJUAvXfx5XA1TdGoAtOCooyeL
+CARTO_CLIENT_SECRET=Hf6N3fucJDUjcX6lYaqUx4bQwr-gp7ch6U3g11bbdwcv0m6gul1Yr8Toa0OU9sb9
```
Comment (srtena) [inline integrate-existing-app/backend/.env.local]: @alasarr oops, not intended, right?

File: private-app/.env [hunk 2]
```diff
@@ -7,4 +7,4 @@ VITE_API_BASE_URL=https://gcp-us-east1.api.carto.com
 
 # Go to app.carto.com -> Developers -> Create new application -> SPA
 # Set this variable if you're building a private application (it needs to create an application.)
-# VITE_CLIENT_ID=XXX
\ No newline at end of file
+VITE_CLIENT_ID=11ANN82SbihAJPgGLWUNaNX0tBXcMHca
```
Comment (srtena) [inline private-app/.env]: same, intended? @alasarr

**Review by srtena — CHANGES_REQUESTED** [review]

File: private-app/.env (hunk 2 above)
Comment (alasarr) [reply private-app/.env]: Yep! Removing it

File: integrate-existing-app/backend/.env.local (hunk 1 above)
Comment (alasarr) [reply integrate-existing-app/backend/.env.local]: `integrate-existing-app/backend/.gitignore` was not excluding the .local I have fixed it. I have also removed those apps as we're in a public repo

**Review by srtena — APPROVED** [review]
✂️ 👌

**Pull request merged**